services/ui-src/src/components/cards/EmailCard.tsx

```diff
@@ -0,0 +1,82 @@
+// components
+import { Flex, Image, Link, Text } from "@chakra-ui/react";
+import { Card } from "components";
+// utils
+import { useBreakpoint } from "utils";
+import { AnyObject } from "types";
+import { createEmailLink } from "utils/other/email";
+// assets
+import spreadsheetIcon from "assets/icons/icon_spreadsheet.png";
+import settingsIcon from "assets/icons/icon_wrench_gear.png";
+
+const iconMap: { [key: string]: { [key: string]: string } } = {
```

Comment:
Okay I'm not sure if this is going to work but I'd love to make this type more definite. If you delete the type annotation here, I'm sure TS will complain on line 33. But if you then change the `Props` type definition on line 50 from `icon: string` to `icon: keyof typeof iconMap`... will there be complaints elsewhere, or will TS be gucci?

```suggestion
const iconMap = {
```